getline() is reserved symbol in libc. rename the app symbol.

--- rxmtp-1.1.3.orig/smtp.c
+++ rxmtp-1.1.3/smtp.c
@@ -54,7 +54,7 @@
 	return (0);
 }
 
-char *getline(char *line, int size, FILE *fp, int debug)
+char *mygetline(char *line, int size, FILE *fp, int debug)
 {
 	*line = 0;
 	if (fgets(line, size, fp) == NULL) { 
@@ -71,7 +71,7 @@
 
 int getresp(char *line, int size, FILE *fp, int debug)
 {
-	if (getline(line, size, fp, debug) == NULL)
+	if (mygetline(line, size, fp, debug) == NULL)
 		return (-1);
 	
 	return (atoi(line));
